docs/dev/contributing.rst

```diff
@@ -31,14 +25,11 @@ Be Cordial
 
 Pipenv has one very important rule governing all forms of contribution,
 including reporting bugs or requesting features. This golden rule is
-"`be cordial or be on your way`_".
+"be cordial or be on your way".
 
 **All contributions are welcome**, as long as
 everyone involved is treated with respect.
 
-.. _be cordial or be on your way: https://www.kennethreitz.org/essays/be-cordial-or-be-on-your-way
```

Comment:
Why do we remove the link?
The valid link would be https://kennethreitz.org/essays/2013/01/27/be-cordial-or-be-on-your-way

Reply:
Great, thank you for finding the correct link.

Reply:
I restored the link.